packages/charts/examples/.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "extends": "../.eslintrc.json",
+  "ignorePatterns": [
+    "!**/*"
```

Comment:
Just out of curiosity what is this for?

Reply:
It is the default value for configured .eslintrc.json file. Will delete it so as not to confuse anyone :)